### PR TITLE
[WIP] Remove deprecated code

### PR DESCRIPTION
**Pull request by emitreva-cb**
## Pull request checklist

Please check if your PR fulfills the following requirements:

- [ ] Docs have been reviewed and added / updated if needed (for bug fixes / features)
- [ ] Tests have been added that prove the fix is effective or that the feature works.
- [ ] New and existing tests pass locally with the changes.
- [ ] Code follows the style guidelines of this project (PEP8, clean code).
- [ ] Linter has passed locally and any fixes were made for failures.
- [ ] A self-review of the code has been done.

## Pull request type

Please check the type of change your PR introduces:
- [ ] Bugfix
- [ ] Feature
- [ ] Code style update (formatting, renaming)
- [ ] Refactoring (no functional changes, no api changes)
- [ ] Build related changes
- [ ] Documentation content changes (not tied to bugs/features)
- [ ] Other (please describe):

## What is the ticket or issue number?

- Ticket Number: N/A

- Issue Number: N/A

## Does this introduce a breaking change?

- [ ] Yes
- [ ] No

### CODE REVIEW

File: examples/response/alert_bulk_resolve.py [hunk 1]
```diff
@@ -1,36 +0,0 @@
-#!/usr/bin/env python
```
Comment (avanbrunt-cb) [inline examples/response/alert_bulk_resolve.py:1]: response should not be deleted as this is EDR I believe

File: src/cbapi/live_response_api.py [hunk 2]
```diff
@@ -1,1214 +0,0 @@
-
-"""The Live Response API and associated objects."""
```
Comment (avanbrunt-cb) [inline src/cbapi/live_response_api.py:2]: Don't get rid of this as EDR can use live response

File: src/cbapi/response/cblr.py [hunk 3]
```diff
@@ -1,52 +0,0 @@
-from cbapi.live_response_api import CbLRManagerBase, CbLRSessionBase, poll_status
```
Comment (avanbrunt-cb) [inline src/cbapi/response/cblr.py:1]: This is EDR don't remove

File: src/cbapi/response/models.py [hunk 4]
```diff
@@ -772,20 +772,6 @@ def resource_status(self):
         """
         return self._cb.get_object("{0}/resourcestatus".format(self._build_api_request_uri()), default=[])
 
-    def lr_session(self):
```
Comment (avanbrunt-cb) [inline src/cbapi/response/models.py:775]: Don't remove as this is EDR

File: src/cbapi/response/rest_api.py [hunk 5]
```diff
@@ -69,15 +68,6 @@ def __init__(self, *args, **kwargs):
 
         self._lr_scheduler = None
 
-    @property
```
Comment (avanbrunt-cb) [inline src/cbapi/response/rest_api.py:72]: Don't remove as this is EDR

File: src/cbapi/psc/__init__.py [hunk 6]
```diff
@@ -1,6 +0,0 @@
-# Exported public API for the Cb PSC API
```
Comment (avanbrunt-cb) [inline src/cbapi/psc/__init__.py:1]: The plan is to remove the entire psc folder right?

**Comment by emitreva-cb** [comment]
I have created a new branch with the changes as it seemed easier then restoring deleted files.

**Pull request closed without merging**